### PR TITLE
[TR] Merge 1.4 into master

### PR DESCRIPTION
**Pull request by phaseinducer**

### CODE REVIEW

File: contributing/conventions.rst [hunk 1]
```diff
@@ -3,15 +3,13 @@ Code Conventions
...
 When contributing code to Akeneo PIM, you must follow its code conventions (very close to Symfony Code conventions).
 
-This is part of our Developer Xperience effort, writing these conventions allows us to share the path we want to follow to continuously improve our codebase.
+This is part of our Developer eXperience effort, writing these conventions allows us to share the path we want to follow to continuously improve our codebase.
 
-There are still inconsistencies in the codebase that we'll fix in the upcoming releases.
+There are still inconsistencies in the codebase that we're fixing in the upcoming minor releases.
 
-We'll continue to enhance the Best Practices section to give clear hints on:
+We'll continue to enhance this section.
```
Comment (rybus) [inline contributing/conventions.rst]: We will keep on enhancing this section.

File: reference/import_export/index.rst [hunk 2]
```diff
@@ -7,7 +7,7 @@ This chapter describes the architecture and behaviour of imports and exports in
 
   The import part has been widely re-worked in 1.4.
 
-  A new system has been introduced, the old system has been deprecated but is kept, both systems are useable in 1.4 but we strongly recommend using the new system.
+  A new system has been introduced, the old system has been deprecated but is kept, both systems are usable in 1.4 but we strongly recommend using the new system.
```
Comment (rybus) [inline reference/import_export/index.rst]: using the new one.

File: reference/import_export/product-import.rst [hunk 3]
```diff
@@ -9,7 +9,7 @@ It's a good start to understand the overall architecture and how to re-use or re
 
   The import part has been widely re-worked in 1.4.
 
-  A new system has been introduced, the old system has been deprecated however is kept, both systems are useable in 1.4 but we strongly recommend using the new system.
+  A new system has been introduced, the old system has been deprecated however is kept, both systems are usable in 1.4 but we strongly recommend using the new system.
```
Comment (rybus) [inline reference/import_export/product-import.rst]: the new one.

File: reference/component-or-bundle.rst.inc [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Akeneo or Pim, Bundle or Component?
+-----------------------------------
+
+Since the 1.3, Akeneo PIM introduced components, they contain business logic or technical logic not coupled with Symfony Framework or Doctrine ORM / MongoDBODM, although Bundles contain the glue to assemble components together.
+
+We use different namespaces, Akeneo for the technical parts not coupled with the PIM, Pim for the Community Edition parts and Enterprise postfix for the Enterprise Edition parts.
+
+.. image:: /reference/component-or-bundle.png
```
Comment (solivier) [inline reference/component-or-bundle.rst.inc]: missing new blank line

**Comment by skeleton** [comment]
GTM

**Comment by nidup** [comment]
baby comments of everybody then GTM :)

**Comment by phaseinducer** [comment]
@nidup : I asked Rémy to open a PR on 1.4 to have the same version everywhere. ; )

File: contributing/conventions.rst [hunk 5]
```diff
@@ -35,26 +34,33 @@ We define them as services, these classes don't extend the BaseController and ar
...
+---------------------------------
+
+We never use ObjectManager persist/remove/flush except in SaverInterface and RemoverInterface.
+
+It allows use to decouple the business code of component from the ORM code.
 
 Services Definition
 -------------------
 
-We define the services in several files depending on the service purpose (form type, manager, etc.).
```
Comment (solivier) [inline contributing/conventions.rst]: :+1:

**Comment by solivier** [comment]
GTM

**Pull request merged**